editors/vscode/server/src/main.rs

```diff
@@ -56,7 +57,7 @@ impl LanguageServer for Backend {
     }
 
     async fn initialized(&self, _: InitializedParams) {
-        self.client.log_message(MessageType::INFO, "oxc initialized.").await;
+        debug!("oxc initialized.");
```

Comment:
log crate could reach the same effort as `self.client.log_message` but use less typing.